docs/release-notes/index.md

```diff
@@ -322,3 +364,10 @@ additional features.
         True to overwrite the existing export path.
     For more information, please refer to
     [Export a scenario](../userman/scenario_features/sdm/scenario/index.md#export-a-scenario).
+
+## Significant bug fixes
+
+<h4></strong> 4.0.1</h4>
+
+- The "taipy" development authentication protocol would drop the user roles when the configuration
+  does not explicitly specify the *protocol* parameter name.
```

Comment:
Ho? I don't recall that. Why? That's strange.

Reply:
That's because the first (unnamed) parameter ("id") was inserted as the first one, breaking compatibility.

Reply:
You are right, this was not a 4.0 issue.